packages/client/lib/AccessTokenClient.ts

```diff
@@ -14,7 +9,12 @@ import {
   IssuerOpts,
   OpenIDResponse,
   PRE_AUTH_CODE_LITERAL,
-} from './types';
+} from '@sphereon/openid4vci-common/lib';
```

Comment:
This is referencing a dir in the package, suggesting the exports are not in order. Should not have to include /lib

Reply:
exports were fine. Intellij auto-refactor upone move added the folder.
Removed the lib part from path.